pkg/azureclients/v2/utils/options.go

```diff
@@ -37,6 +37,7 @@ func GetDefaultOption(apiVersion string) *arm.ClientOptions {
 				custompolicy.NewThrottlingPolicy(),
 			},
 			APIVersion: apiVersion,
+			Transport:  defaultHTTPClient,
```

Comment:
does it work with https://github.com/Azure/go-armbalancer?

Reply:
added go-armbalancer.